src/Agent/NewRelic/Agent/Core/Configuration/AppSettingsConfigResolveWhenUsed.cs

```diff
@@ -79,6 +80,10 @@ public static string GetAppSetting(string key)
                 }
                 else
                 {
+                    if (key.Equals(Constants.AppSettingsLicenseKey))
+                    {
+                        value = Strings.ObfuscateLicenseKey(value);
```

Comment:
Would it be better for us to log a similar message to the one when the license key is found in an environment variable?

Reply:
I think that logging at least part of the license key is useful for troubleshooting; it confirms to the customer that the right key is being set (unless they mangle some part of the key after the first 8 chars).

Reply:
We should probably be consistent across the different ways we encounter the license key, but I think either a message or obfuscation is fine.